domain/schema/model.go

```diff
@@ -454,6 +455,64 @@ CREATE TABLE cloud_container (
 
 CREATE UNIQUE INDEX idx_cloud_container_net_node
 ON cloud_container (net_node_uuid);
+
+CREATE TABLE instance_data (
+    uuid            	   TEXT PRIMARY KEY,
+    machine_uuid    	   TEXT NOT NULL,
+    instance_id 	   TEXT NOT NULL,
+    display_name 	   TEXT NOT NULL,
+    arch            	   TEXT,
+    mod 	    	   INT,
```

Comment:
What's `mod` in this case?

Reply:
sorry, it was `mem`, I don't know why I typed that :rofl:, or maybe a bad replace